addons/sourcemod/scripting/surftimer/convars.sp

```diff
@@ -364,6 +366,9 @@ void CreateConVars()
 
 	HookConVarChange(g_hEnforceDefaultTitles, OnSettingChanged);
 
+	// SaveLoc
+	g_hallowCheckpointRecreation = AutoExecConfig_CreateConVar("ck_allow_checkpoint_recreation", "0", "Allow player checkpoint recreation (saveloc). 0 - Disabled | 1 - Print info to player chat | 2 - Print info to player console");
```

Comment:
can we have 3rd option where we send msg to both chat and console?

Reply:
Done